uktrade/data-hub-frontend
# Use hawk auth for help centre feed

## Description of change

This PR fixes a long standing bug which has caused the help centre feed to not appear on the homepage.

This was caused by a bug in the help centre service itself, which has now been fixed by allowing hawk authentication to connect to its api.

As a result, this PR updates the request made to the help centre API feed to use hawk.

**TO NOTE**
The previous implementation passed a 1000 miliseconds timeout in the request to the help centre API, this stops the homepage hanging if the help centre api is unavailable. To ensure this functionality is kept, I have updated the axios hawk request module to take an optional timeout parameter.

## Test instructions

- Get `HELP_CENTRE_ACCESS_KEY_ID` and `HELP_CENTRE_SECRET_ACCESS_KEY` values from either dev or staging vault
- Add them to your local env vars
- You should see a recent article on your homepage

## Screenshots
### Before
![Screenshot 2021-12-14 at 15 15 04](https://user-images.githubusercontent.com/22460823/146025617-2207a195-a649-4d59-8dc9-d21915d8cdcc.png)


### After
![Screenshot 2021-12-14 at 15 14 38](https://user-images.githubusercontent.com/22460823/146025536-a87cc0bd-af9f-4900-9d7b-4532b7ca0c5c.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
